static/src/javascripts/projects/common/modules/commercial/acquisitions-link-enrichment.js

```diff
@@ -51,16 +50,10 @@ const addReferrerDataToAcquisitionLinksOnPage = (): void => {
     const links = [...document.getElementsByClassName(ACQUISITION_LINK_CLASS)];
 
     links.forEach(el => {
-        fastdom.read(() => el.getAttribute('href')).then(link => {
-            if (link) {
-                fastdom.write(() => {
-                    el.setAttribute(
-                        'href',
-                        addReferrerDataToAcquisitionLink(link)
-                    );
-                });
-            }
-        });
+        const link = el.getAttribute('href');
```

Comment:
tip: It's marginally faster to access properties instead of attributes, so `el.pathname` or `el.href` depending on the use case.